src/modules/Experiences/AddOrEditExperienceForm/AddOrEditExperienceForm.js

```diff
@@ -33,17 +36,25 @@ const useForm = (initialValues, onSubmit) => {
     handleDegree: (event) => {
       update('degree')(event.target.value);
     },
+    handleEndDateNone: (e) => {
```

Comment:
This name is confusing, I would suggest the simpler `handleIsCurrent`

Reply:
done